Qfront/src/redux/product/loadProducts/actions.js

```diff
@@ -0,0 +1,57 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import { baseUrl, getAllCategoriesApi, productsEndPointApi
+} from "../../api";
+
+// Create action to load by price
+export const getProductsByPrice = createAsyncThunk("LOAD_BY_PRICE", async( _, { rejectWithValue }) => {
+    try {
+       const getProductsByPriceApi = `${productsEndPointApi}?sortBy=price&order=asc&limit=10`;
```

Comment:
We should avoid hardcoding moving forward. We can have these as default values for the thunk's parameters rather than hard coding.